### PR TITLE
Merge pull request #3 from timmolter/develop

### PR DESCRIPTION
**Pull request by cacofishadct**
sync

### CODE REVIEW

**Comment by timmolter** [comment]
can you explain this PR? Seems weird.

**Comment by cacofishadct** [comment]
I suppose you are referring to the changing limit to String part?
from https://www.kraken.com/help/api. 
limit = maximum net amount that can be deposited right now, or false if no limit
The old code will throw error since it may return false and failed to parse as BigDecimal.

**Comment by cacofishadct** [comment]
Was trying to sync originally in the PR, but later added more changes to the code.

File: xchange-kraken/src/main/java/org/knowm/xchange/kraken/dto/account/KrakenDepositAddress.java [hunk 1]
```diff
@@ -5,13 +5,16 @@
...
   private final Long expiretm;
   private final boolean newAddress;
 
-  public KrakenDepositAddress(@JsonProperty("address") String address, @JsonProperty("expiretm") Long expiretm,
+  public KrakenDepositAddress(@JsonProperty("address") String address,
+      @JsonProperty("tag") String tag, @JsonProperty("expiretm") Long expiretm,
       @JsonProperty("new") boolean newAddress) {
     super();
     this.address = address;
+    this.tag = tag;
```
Comment (timmolter) [inline xchange-kraken/src/main/java/org/knowm/xchange/kraken/dto/account/KrakenDepositAddress.java]: I don't see the tag field in the API docs: https://www.kraken.com/help/api#private-user-funding

**Comment by timmolter** [comment]
OK, I see that limit can be false now, according to their API docs.

File: xchange-kraken/src/main/java/org/knowm/xchange/kraken/dto/account/KrakenDepositAddress.java (hunk 1 above)
Comment (cacofishadct) [reply xchange-kraken/src/main/java/org/knowm/xchange/kraken/dto/account/KrakenDepositAddress.java]: True. It's not in the doc.  I tested out....

Comment (timmolter) [reply xchange-kraken/src/main/java/org/knowm/xchange/kraken/dto/account/KrakenDepositAddress.java]: Are you saying `tag` is in the response based on your experience?

Comment (cacofishadct) [reply xchange-kraken/src/main/java/org/knowm/xchange/kraken/dto/account/KrakenDepositAddress.java]: Yes, I tried 2 coins which need tag/memo, and tag is always returned in that case.

**Pull request merged**